client/Comms.js

```diff
@@ -3,7 +3,8 @@ function Comms() {
 	this.outgoing = []
 	this.incoming = []
 	this.listeners = []
-
+	this.lastConnection = []
```

Comment:
It is not. I took it out on my local version, just didn't get around to committing.